### ISSUE TITLE
ESP32 yaml with internal pullup + 2 new sensors

### ISSUE THREAD

**Issue by Chreece** [tsunglung/esphome-ld2450#3]
Thank you very much for your great work implementing the LD2450 on ESPHome.
I wanted to inform you that I slightly edited the .yaml file to use an ESP32 with internal pullup resistors (no external resistors needed).
Also added 2 software sensors: 

- Presence detected
- Number of targets detected

https://github.com/Chreece/esphome-ld2450

**Comment by DerPicknicker** [comment]
Would it be a lot effort to implement zones directly in esphome? And a map which visualise it in Homeassistant?

**Comment by Raimon93** [comment]
@Chreece  while trying ur yaml code i got an error due to those targets id are not existing? which i indeed can not find back somewhere?

**Comment by DerPicknicker** [comment]
It's not my code .. @Chreece is the right one

**Comment by Raimon93** [comment]
> It's not my code .. @Chreece is the right one

Yea sorry already replaced the tagging!

**Comment by Chreece** [comment]
> @Chreece while trying ur yaml code i got an error due to those targets id are not existing? which i indeed can not find back somewhere?

Yeah sorry for that, my setup has more than one sensors so I kinda copied a bit from my setup and a bit from original code, forgetting some parts... Can you please try it now?

*Also from now on, please report any issues on my repository

**Comment by Chreece** [comment]
> Would it be a lot effort to implement zones directly in esphome? And a map which visualise it in Homeassistant?

Unfortunately my ESPHome knowledge is not expert, so I don't know if there is a way for ESPHome to pass a map to HA. You have the coordinates from targets, how one can use them is another story...

**Comment by Raimon93** [comment]
@Chreece Ah now i understand that id(targetx). Thanks!
